docs/paper.md

```diff
@@ -94,16 +94,18 @@ In order to meet the needs of researchers, `scores` provides the following key b
 
 **Compatability**
 
-- Highly modular and avoids extensive dependencies by providing its own implementations where relevant.
-- Easy to integrate and use in a wide variety of environments. It has been tested and used on workstations, servers and in high performance computing (supercomputing) environments.
+- Highly modular - provides its own implementations, avoids extensive dependencies and allows greater API consistency.
+- Easy to integrate and use in a wide variety of environments. It has been used on workstations, servers and in high performance computing (supercomputing) environments. 
+- Maintains 100% automated test coverage.
```

Comment:
You already mentioned 100% test coverage above on line 64. I suggest that it only needs mentioning once.

Reply:
Agreed, thanks